File: pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get input
    int input;
    do
    {
        input = get_int("Height: ");
    }
    while (input < 1 || input > 8);

    //print blocks
    for (int i = 1; i <= input; i++)
    {
        for (int x = 0; x < input; x++)
        {
            //return space if x(colum) is less than input - row
            printf(x < input - i ? " " : "#");
        }
        printf("\n"); //print new line at the end of the row
    }

}

File: pset6/mario/more/mario.py
from cs50 import get_int

# check if value is between 1 and 8 and if is decimal number
while True:
    inp = input("Height: ")
    if str.isdecimal(inp) and int(inp) in range(1, 9):
        break

height = int(inp)

# print the blocks
for i in range(1, height + 1):
    print(f"{' ' * (height - i)}{'#' * i}  {'#' * i}")
